### PR TITLE
Add CI via github actions

### PR DESCRIPTION
**Pull request by yudistrange**

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
```
Comment (kitallis) [inline .github/workflows/elixir.yml:9]: Let's format this a bit better, like [this](https://github.com/inclojure-org/inclojure-org.github.io/blob/trunk/.github/workflows/pages.yml).

File: .github/workflows/elixir.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Elixir CI
```
Comment (kitallis) [inline .github/workflows/elixir.yml:1]: This is neat and simple. Shall we also add the build step in this PR itself?

Comment (yudistrange) [reply .github/workflows/elixir.yml:1]: It's added to the PR step as well. The `pull_request` key dictates that afaik.

**Review by kitallis — APPROVED** [review]
Looks good to me ✨

**Pull request merged**